src/operator/tensor/elemwise_binary_scalar_op_basic.cc

```diff
@@ -53,16 +53,16 @@ static bool BinaryScalarStorageTypeWithDenseResultStorageType(const NodeAttrs& a
                                                               std::vector<int>* in_attrs,
```

Comment:
Since the result is not always dense now for `BinaryScalarStorageTypeWithDenseResultStorageType`, we should add some comment and clarify that so that there's no misuse.

Reply:
Added